pkg/pool-manager/pool.go

```diff
@@ -41,6 +41,7 @@ const (
 	networksAnnotation             = "k8s.v1.cni.cncf.io/networks"
 	networksStatusAnnotation       = "k8s.v1.cni.cncf.io/networks-status"
 	transactionTimestampAnnotation = "kubemacpoolTransactionTimestamp"
+	noPendingTransaction           = "no-pending-transaction"
```

Comment:
is `macEntryNotPendingChange` good?

Reply:
I ended up removing this var since I changed the timestamp to be *time.Time where nil means noPendingTransaction, and since I can't save a const var with nil ([link](https://stackoverflow.com/questions/57554386/why-can-a-const-initializer-not-be-nil))